### ISSUE TITLE
Enlever "ajouter-là" dans la page de recherche /Main/Search

### ISSUE THREAD

**Issue by JeanPerrier** [DISIC/wikidemarches#515]
Il faudrait enlever le bouton ajouter une démarche dans le moteur de recherche.

**Comment by lucaa** [comment]
Je ne comprends pas, on enlève complètement la possibilité d'ajouter une démarche manquante?

**Comment by JeanPerrier** [comment]
https://monavis.numerique-staging.devxwiki.com//Main/Search
C'est l'avis de recherche de cerfa numérique. Sur cette page, on te propose d'ajouter une démarche, hors tu n'es pas authentifié. 
Or, normalement, pour ajouter une démarche, il faut être authentifier.

**Comment by lucaa** [comment]
> Or, normalement, pour ajouter une démarche, il faut être authentifier.

Effectivement, c'est pour ça que lors du click sur ce bouton on demande l'authentification et on propose la création d'un compte. Il n'y a donc pas de trou de sécurité.

En fait, j'ai pensé l'affichage du bouton pour utilisateurs non-authentifiés comme une fonctionnalité, pas comme un bug, ainsi:
* si la démarche d'un utilisateur cerfa n'est pas encore dans le référentiel, ce bouton lui montre qu'il peut toujours ajouter sa démarche mais qu'il doit se créer un compte avant. 
Cette information est d'autant plus importante que l'ajout d'une nouvelle démarche était une action ouverte au public en versions < M8.
